<-PYT:import urllib2
<-PYT:urllib2.urlopen('http://bugzilla.myveryoldsite.com/cli.cgi?bug=42')
->RET:raise Instance('URLError', 'urlerror1')
<-PYT:urlerror1.__str__()
->RET:"<urlopen error (-2, 'Name or service not known')>"
